docs/pages/setup/helm-reference/teleport-cluster.mdx

```diff
@@ -415,6 +415,119 @@ $ kubectl --namespace teleport create secret generic license --from-file=/path/t
   </TabItem>
 </Tabs>
 
+## `installCRDs`
+
+| Type | Default value | Can be used in `custom` mode? |
+| - | - | - |
+| `bool` | `false`| ✅ |
+
+CRDs are required by the teleport-operator and are installed by default when `operator.enabled` is true.
+`installCRDs` overrides this behaviour and allows users to explicit whether to deploy Teleport CRDs.
+
+Unless you are deploying multiple `teleport-cluster` Helm releases in the same Kubernetes cluster or installing
+the CRDs on your own you should not have to set this value.
+
+<Admonition type="note">
```

Comment:
We should break the text out of this Admonition box and move it higher up in the explanation of this field. Otherwise, we need to navigate to the end of this reference entry before we find out why this field exists.

Reply:
done